src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+---
+title: What is Broccoli?
+---
+
+Broccoli is a simple JavaScript build tool, that allows you to configure your build pipeline in JavaScript (like
+you're already used to writing), and handles setting up the filesystem state for each transformation that's 
+going to happen.
+
+${toc}
+
+## What is a build tool?
+
+A build tool's job is to take input files (your javascript, css, html, etc), to process them and output the result
+of those transformations, usually into some distributable form. Typically this will involve things like JavaScript 
+transformations to allow you to write newer syntax that will work in a browser, to use a CSS pre-processor like 
+Sass for your CSS, etc.
+
+Broccoli.js is different to other build tools. You may be used to tools like [Grunt](https://gruntjs.com/) (a task
+runner), [Gulp](https://gulpjs.com/) (streams and pipes) or [Webpack](https://webpack.js.org/) (a module bundler),
+these all operate on different levels to Broccoli.
+
+Broccoli works at the file-system level, it provides a JavaScript API to wrap other node tools like
+[Babel](https://babeljs.io/), [Rollup](https://rollupjs.org/) or [Node-Sass](https://github.com/sass/node-sass), 
+[Webpack](https://webpack.js.org/), [Browserify](http://browserify.org/), and many more. These tools operate on a
+set of input files, and emit output files, that is all. Broccoli has no knowledge of the contents of your 
+files, only input and output directories.
+
+## Thinking in Broccoli
+
+Broccoli at its heart is very simple. It is really just a set of directories, connected together by a plugin
+system. This plugin systems allow behavior between input and output directories to be described. Broccoli then
+ensures plugins are called at the appropriate time, to ensure the build is successful, and writes the result to
+a `target` directory. Each plugin is responsible for processing files passed to it in input directories, and 
+writing files to its output directory.
+
+Broccoli doesn't really care about files, it simply takes source directories and passes them as inputs to 
+plugins, creates an output directory for the plugin to write to, and passes that output directory it as an input
+to the next plugin.
```

Comment:
You're taking a lot of time to try and describe something that might be easier to show via a diagram

```
app                                       dist
 └─ src						         	   └─ prod.build-20190206.js
     ├─ index.ts      -->  MyPlugin -->
     └─ other.ts

```